packages/storybook-builder/src/index.ts

```diff
@@ -1,7 +1,8 @@
 import rollupPluginNodeResolve from '@rollup/plugin-node-resolve';
 import { getBuilderOptions } from '@storybook/core-common';
 import { logger } from '@storybook/node-logger';
-import { globals } from '@storybook/preview/globals';
+// @ts-ignore
+import { globals, globalsNameReferenceMap } from '@storybook/preview/globals';
```

Comment:
wouldnt just the `globalsNameReferenceMap` be enough? Why are we using both?

Reply:
for retrocompatibility. Those using 7.5.3 or lower will have an error if we don't support the previous behaviour.

Reply:
Ah I see. Could you add a comment to the import that we import both for this reason? Just so we'll know why this is in the future :)

Reply:
Done

Reply:
Thanks :) just running CI to make sure everything passes, then we can merge it

Reply:
Yup, fixed linting problems